### DISCUSSION TITLE
Creating a musikcube client for iOS

### DISCUSSION THREAD

**Discussion by dbolger** [clangen/musikcube#513] [Ideas]
I'm interested in developing for a client for iOS. What are some considerations I should keep in mind while writing this? Where can I find the musikdroid source?

I don't imagine writing this client to be challenging since it just uses an HTTPd client to host the audio.

**Comment by henriquenunez** [comment]
Hey, interested as well! Any updates here?

**Reply by dbolger** [reply]
I put a halt on the project as I don't have a device with macOS to develop on.
